.buildkite/scripts/steps/fips/smoke_test.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+if [ -z "$KIBANA_BUILD_LOCATION" ]; then
+  export KIBANA_BUILD_LOCATION="/usr/share/kibana"
+fi
+
+# a FTR failure will result in the script returning an exit code of 10
+exitCode=0
+
+configs=(
+  "x-pack/test/reporting_functional/reporting_and_security.config.ts"
+  "x-pack/test/saved_object_api_integration/security_and_spaces/config_trial.ts"
+  "x-pack/test/alerting_api_integration/security_and_spaces/group1/config.ts"
+  "x-pack/test/alerting_api_integration/security_and_spaces/group2/config.ts"
+  "x-pack/test/alerting_api_integration/security_and_spaces/group3/config.ts"
+  "x-pack/test/alerting_api_integration/security_and_spaces/group4/config.ts"
+  "x-pack/test/functional/apps/saved_objects_management/config.ts"
+  "x-pack/test/functional/apps/user_profiles/config.ts"
+  "x-pack/test/functional/apps/security/config.ts"
+)
+
+cd /home/vagrant/kibana
+
+for config in "${configs[@]}"; do
+  set +e
+  node /home/vagrant/kibana/scripts/functional_tests \
+    --bail \
+    --kibana-install-dir "$KIBANA_BUILD_LOCATION" \
+    --config="$config"
+  lastCode=$?
+  set -e
+
+  if [ $lastCode -ne 0 ]; then
+    exitCode=10
+    echo "FTR exited with code $lastCode"
+    echo "^^^ +++"
+
+    if [[ "$failedConfigs" ]]; then
+      failedConfigs="${failedConfigs}"$'\n'"$config"
+    else
+      failedConfigs="$config"
+    fi
+  fi
+done
+
+if [[ "$failedConfigs" ]]; then
+  buildkite-agent meta-data set "$failedConfigs"
```

Comment:
This is failing since the script runs inside the VM box and the BK functions are not available. Not sure if we can pass those along from the host agent.